grafanalib/core.py

```diff
@@ -299,6 +302,61 @@ def TimePicker(refreshIntervals, timeOptions):
 )
 
 
+def AlertCondition(target, queryParams=["A", "5m", "now"],
+                   evaluator=["gt", 5], operator="and", type="avg"):
+    """
+    Alert conditions:
+        :param target: Target
+        :param queryParams: [refId, in_time, "now"]
+        :param evaluator: [gt/lt/within_range/outside_range/no_value, values]
```

Comment:
Can you please define constants for these (similar to the ones we use for axes, e.g. `SHORT_FORMAT`), and use those in the docstring and the function body?